tests/test_autocommit.py

```diff
@@ -0,0 +1,11 @@
+import sys, os
+
+def test():
+    "Verify autocommit just before program exits."
+    assert(0 == os.system('PYTHONPATH=. %s tests/autocommit.py' % sys.executable))
+    # The above script relies on the autocommit feature working correctly.
+    # Now, let's check if it actually worked.
+    import sqlitedict
+    d = sqlitedict.SqliteDict('tests/db/autocommit.sqlite')
+    for i in range(1000):
+        assert d[i] == i, [d[i], i]
```

Comment:
If you're going to leave an assertion message, then make it a bit more helpful:

```suggestion
        assert d[i] == i, "actual: %d expected: %d" % (d[i], i)
```

Reply:
Is there any advantage to using `%d` over `%s`? The `%s` seems more robust and future-proof.

Reply:
Yeah, '%s' is marginally better in this case, because it won't trigger an exception when generating the assertion message.